### PR TITLE
[MM-51467] - NotifyAdmin job reports an error for unlicensed servers

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
Problem
---
- The scheduler for the job that sends notifications about plugins to install was running too often i.e. every 1 minute or so.
- The function that sends the notifications, `DoCheckForAdminNotifications` has a check for getting the current license and will error if that license is not found. The reason a license check was added here was to get the current sku and then filter out any notifications to the admin in case the admin had already upgraded to the current sku from the license. As the notify admin feature got used for other notifications like installing plugins even on self-hosted installations where starter has no license attached, the function `DoCheckForAdminNotifications` will error

Solution
---
- Reduce scheduler run frequency to every 24 hours
- Remove an error that occurs because license was not found. Instead default the current sku to starter and if a license is found, pick the sku from the license. All callers of `DoCheckForAdminNotifications` should be incharge of making a more strict check for a license if they need to, for example https://github.com/mattermost/mattermost-server/blob/63e7c29343075db9fa673a90ddc0cb88050beb1f/jobs/notify_admin/scheduler.go#L19

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-51467

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by neallred — APPROVED** [review]
LGTM, thanks @AGMETEOR !

File: app/notify_admin.go [hunk 1]
```diff
@@ -48,12 +48,12 @@ func (a *App) SaveAdminNotification(userId string, notifyData *model.NotifyAdmin
...
+	currentSKU := "starter"
 	license := a.Srv().License()
-	if license == nil {
-		return model.NewAppError("DoCheckForAdminNotifications", "app.notify_admin.send_notification_post.app_error", nil, "No license found", http.StatusInternalServerError)
+	if license != nil {
+		currentSKU = license.SkuShortName
 	}
 
-	currentSKU := license.SkuShortName
 	workspaceName := ""
```
Comment (nickmisasi) [inline app/notify_admin.go:57]: is this supposed to always be an empty string?

**Comment by amyblais** [comment]
@AGMETEOR Will this need a QA review?

**Comment by AGMETEOR** [comment]
> @AGMETEOR Will this need a QA review?

I don't think so

**Comment by amyblais** [comment]
@AGMETEOR Please help open manual cherry-pick PRs for ``cloud`` and ``release-7.9`` branches.

**Pull request merged**